### ISSUE TITLE
Invalid Escape Sequence Deprecation Warning

### ISSUE THREAD

**Issue by Yash-10** [astropy/ccdproc#751]
Hello,

While testing using `pytest`, there are a bunch of warnings that pop up. One of the error is this:

```
DeprecationWarning: invalid escape sequence \_
    """
```
This is because of a `\` in the docstring of the function `bitfield_to_boolean_mask` [here](https://github.com/astropy/ccdproc/blob/master/ccdproc/extern/bitfield.py#L249) in and I observed that converting that docstring to a rawstring (`r"""......"""`) resolves that issue. This is also suggested as per PEP 257 Docstring Conventions as:

> For consistency, always use """triple double quotes""" around docstrings. Use r"""raw triple double quotes""" if you use any backslashes in your docstrings.

Maybe this can be included?

Thanks!

Version specifications:
```
>>> print(astropy.__version__)
4.2
>>> print(ccdproc.__version__)
2.1.1.dev29+g6ee2f67.d20210108
>>> print(numpy.__version__)
1.19.5
>>> pytest.__version__
'6.2.1'
```

**Comment by Yash-10** [comment]
@mwcraig Does this look good to be incorporated?

Thanks!

**Comment by mwcraig** [comment]
Closed by #759
